### PR TITLE
Added applyEulerFilter to import

### PR DESCRIPTION
**Pull request by tryptik**
It has proven useful to allow the Euler filtering of rotational animation on import as well as export. This P/R adds a flag `aef/applyEulerFilter` flag to `mayaUSDImport` that applies Euler filtering for transform and skeletal animation data.

### CODE REVIEW

File: lib/mayaUsd/fileio/jobs/jobArgs.cpp [hunk 1]
```diff
@@ -1333,8 +1336,8 @@ std::ostream& operator<<(std::ostream& out, const UsdMayaJobImportArgs& importAr
         << std::endl
         << "timeInterval: " << importArgs.timeInterval << std::endl
         << "useAsAnimationCache: " << TfStringify(importArgs.useAsAnimationCache) << std::endl
-        << "preserveTimeline: " << TfStringify(importArgs.preserveTimeline) << std::endl
```
Comment (pierrebai-adsk) [inline lib/mayaUsd/fileio/jobs/jobArgs.cpp:1336]: Why did we lose preserve timeline here?

**Review by pierrebai-adsk — APPROVED** [review]
Great work

**Review by pierrebai-adsk — DISMISSED** [review]

**Comment by seando-adsk** [comment]
@tryptik Unfortunately the clang-format step (of the preflight) failed. Can you please make the changes and I'll re-run the preflight again.

**Comment by tryptik** [comment]
I have run the clang formatter and removed some older code fragments from a previous implementation.

**Review by pierrebai-adsk — DISMISSED** [review]

**Comment by seando-adsk** [comment]
@tryptik Unfortunately the clang-format step failed again. If you are running it locally make sure you are using v10.0.0.

**Review by pierrebai-adsk — APPROVED** [review]

**Pull request merged**